stats.js

```diff
@@ -0,0 +1,55 @@
+
+
+const fs = require("fs")
+const fetch = require('node-fetch');
+
+//let data = fs.readFileSync('./new.md',"utf-8");
+
+const getLinks = (data)=>{
+    let urlRegex = /(https?:\/\/[^\s)]+)/gi;
+    let links = data.match(urlRegex)
+    return links
+}
+//let links = getLinks(data)
+
+const uniqueLinks = (links) => {
+    let repeted= 0;
+    let i;
+    let j=i+1;;
+    for(i=0; i<links.length; i++ ) {
+        for(j=i; j<links.length; j++ ) {
+            if(i != j){
+                if(links[i] == links[j]){
+                    repeted ++;
+                }
+            }    
+        }
+            
+    }
+    let unique = links.length - repeted;
+    return "Unique:" + unique;
+}
+//console.log(uniqueLinks(links))
+
+let ok = 0;
```

Comment:
definir mejor el nombre
```suggestion
let linksOk = 0;
```